core/templates/domain/creator_dashboard/creator-dashboard-stats.model.ts

```diff
@@ -16,9 +16,6 @@
  * @fileoverview Frontend domain object factory for creator dashboard stats.
```

Comment:
Update the fileoverview (i.e, this file in so more a factory service)

Follow this comment in all other files*

Reply:
done here and elsewhere!